### ISSUE TITLE
Linking to open-to-all community pages broken. Forces user to Riot Bot page. 

### ISSUE THREAD

**Issue by vesterinen** [vector-im/riot-web#6247]
### Description

When trying to direct people to my open-to-all community page, it pushes the person to the Riot Bot page instead of my community page (asking a username again, this time for the matrix homeserver). 

### Steps to reproduce

- The first username prompt comes before riot loads anything (the background is white)
- Then it loads the community, opens a new username prompt and switches to riot-bot. 
- If you close the second prompt and press back, it shows the community page.
- When it should have asked the username only once and then showed my community page

<img width="996" alt="screenshot 2018-02-24 12 34 39" src="https://user-images.githubusercontent.com/980486/36635568-462a70a0-196c-11e8-965e-7f07917f3f0c.png">

### Version information

- **Platform**: web (in-browser). 

For the web app:

- **Browser**: Chrome Version 64.0.3282.167 (Official Build) (64-bit), and Safari Version 11.0.3 (13604.5.6). Both Safari and Chrome private modes so it would show the experience for a user without account.
- **OS**:macOS
- **URL**: https://riot.im/app/

**Comment by lampholder** [comment]
We haven't optimised for communities' providing a linkable landing page for brand new, previously unregistered users (as I think is the case here), though doing that work would be a huge win (and obviously it's disappointing that the current impl doesn't allow new users to land on communities pages even if it doesn't automatically sign them up to the group on completion of registration).

My sense is that this should be an ASAP-after-0.14 item.

**Comment by vesterinen** [comment]
Thanks for the quick response. Understand if this is not the intended design but for me it's critical and important given the open community I hope to host. The smoother the experience for the new user the better - Ideally they would get as clear and unobstructed view of the community even if they are not part of it yet. This is to introduce them to what the community is about and avoid them to bounce just because they don't feel like going through a registration flow before they get a sense of what they are looking at. Once they want to contribute I'm sure they don't have a problem to signup. Thanks again for being very responsive. Big fan of Riot!

**Comment by turt2live** [comment]
This is a duplicate of #5650

**Comment by t3chguy** [comment]
Closing as duplicate
